indra/tests/test_eidos.py

```diff
@@ -283,6 +283,15 @@ def test_get_agent_bio():
         assert agent.db_refs['TEXT_NORM'] == norm_text
 
 
+def test_bio_custom_grounding():
+    def my_grounder(txt, context):
+        return {'MYDB': 'MYGROUNDING'}
```

Comment:
are these dictionaries always `Mapping[str, str]`? That would improve the type annotation over `Mapping[str, Any]`